mappings/net/minecraft/server/command/SpreadPlayersCommand.mapping

```diff
@@ -55,6 +56,9 @@ CLASS net/minecraft/class_3131 net/minecraft/server/command/SpreadPlayersCommand
 		ARG 15 respectTeams
 	METHOD method_29193 (Lcom/mojang/brigadier/context/CommandContext;)I
 		ARG 0 context
+	METHOD method_39991 (Ljava/lang/Object;Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 maxHeight
+		ARG 1 minimumY
```

Comment:
`minimumY` might be misinterpreted as the minimum Y of the range the entities are spread over.

```suggestion
		ARG 1 worldBottomY
```

Reply:
if we use maxY above then this should match and be minY (assuming they are such functionality wise-

Reply:
This is a lambda used in the exception for when the 'max height' argument of the command is a smaller number than the world's bottom Y value.